src/Diagnostics/src/LogFile.cpp

```diff
@@ -0,0 +1,56 @@
+#include "OpenLoco/Diagnostics/LogFile.h"
+#include <fmt/chrono.h>
+#include <fmt/format.h>
+#include <fmt/ostream.h>
+
+namespace OpenLoco::Diagnostics::Logging
+{
+    LogFile::LogFile(const fs::path& file)
+    {
+        // Ensure the directory exists in case the filepath is a relative path and contains a sub directory.
+        fs::create_directories(file.parent_path());
```

Comment:
Unfortunately this is not the first thing that happens that uses the logs. `cleanupLogFiles` will be called before this and crash because the log folder does not exist.

Reply:
That's now guarded with try/catch, this should be the first place to create the directory for when the log file is requested.